molecule/mysql.galera.4.mysql_wsrep.8.0/prepare.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- hosts:
+    - debian.buster
+    - debian.bullseye
+  tags: [always]
+  tasks:
+    - name: Apt
+      import_role:
+        name: manala.roles.apt
+      vars:
+        manala_apt_preferences:
+          - galera@galera_4_10
+          - mysql@mysql_wsrep_8_0_26
```

Comment:
i wonder if the test should be called `mysql.galera.4.10.mysql_wsrep.8.0.26` instead of just `mysql.galera.4.mysql_wsrep.8.0`...
What do you think ?